src/librbd/api/Group.cc

```diff
@@ -731,7 +731,7 @@ int Group<I>::snap_create(librados::IoCtx& group_ioctx,
       ldout(cct, 1) << "Failed to create io context for image" << dendl;
     }
 
-    ldout(cct, 20) << "Openning image with id" << image.spec.image_id << dendl;
+    ldout(cct, 20) << "Openning image with id " << image.spec.image_id << dendl;
```

Comment:
Nit: can you correct the spelling of “opening” while you are changing this line?